src/Pages/Overview/Overview.css

```diff
@@ -72,7 +72,7 @@ thead.td{
   box-sizing: border-box;
   font-size: 16px;
   initial-letter-align: left
-}
+} */
```

Comment:
if everything above this line is commented out remove it.

Reply:
weird... I didn't touch this file at all